packages/slate/src/models/text.js

```diff
@@ -16,9 +14,10 @@ import memoize from '../utils/memoize'
  */
 
 const DEFAULTS = {
-  characters: new List(),
+  leaves: List(),
   key: undefined,
 }
+const emptyLeaf = Leaf.create({})
```

Comment:
What's this doing here? Why is it not in `CONSTANT_CASE`? Why does it not have a docblock comment? Why is there no line break before it?

Reply:
Fixed~ It is a pre-created Leaf to speed up empty text Leaf creation.